src/main/java/org/jenkinsci/plugins/stashNotifier/StashNotifier.java

```diff
@@ -530,7 +531,8 @@ protected CloseableHttpClient getHttpClient(PrintStream logger, Run<?, ?> run, S
         RequestConfig.Builder requestBuilder = RequestConfig.custom()
                                             .setSocketTimeout(timeoutInMilliseconds)
                                             .setConnectTimeout(timeoutInMilliseconds)
-                                            .setConnectionRequestTimeout(timeoutInMilliseconds);
+                                            .setConnectionRequestTimeout(timeoutInMilliseconds)
```

Comment:
According to usage search, the function isn't used anymore and deprecated. Thus, instead of adding tests to these changes, I can drop them. What do you think?

Reply:
It seems low risk so I think we can leave it. If there is an issue we can revert.